packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -1028,12 +1033,9 @@ describeCompat("IdCompressor Summaries", "NoCompat", (getTestObjectProvider, com
 		// attached data store.
 		await ds2.trySetAlias("foo");
 
-		assert(
-			// For some reason TSC gets it wrong - it assumes that attachState is constant and that assert above
-			// established it's AttachState.Detached, so this comparison is useless.
-			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-			// @ts-ignore
```

Comment:
I just tried removing it and nothing complained. Might have been a transient issue that a TS server restart would have gotten rid of?

Reply:
`assert.equal` isn't strongly typed, so presumably that's the difference here.